sdk/keyvault/keyvault-admin/README.md

```diff
@@ -1,30 +1,29 @@
-# Azure Key Vault Admin client library for JavaScript
+# Azure Key Vault Administration client library for JavaScript
 
 Azure Key Vault is a service that allows you to encrypt authentication keys, storage account keys, data encryption keys, .pfx files, and passwords by using secured keys.
-If you would like to know more about Azure Key Vault, you may want to review: [What is Azure Key Vault?](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-overview)
+If you would like to know more about Azure Key Vault, you may want to review: [What is Azure Key Vault?][docs-overview]
 
 The package `@azure/keyvault-admin` provides support for the administrative Key Vault tasks. It includes the backup and restore operations for whole Key Vault instances, and the role-based access control (RBAC) operations.
 
-[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-admin) | [Package (npm)](https://www.npmjs.com/package/@azure/keyvault-admin) | [API Reference Documentation](https://docs.microsoft.com/javascript/api/@azure/keyvault-admin) | [Product documentation](https://azure.microsoft.com/en-us/services/key-vault/) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-admin/samples)
+[Source code][package-gh] | [Package (npm)][package-npm] | [API Reference Documentation][docs] | [Product documentation][docs-service] | [Samples][samples]
 
 ## Getting started
 
-**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/) and a
-[Key Vault resource](https://docs.microsoft.com/en-us/azure/key-vault/quick-create-portal) to use this package.
-
-If you are using this package in a Node.js application, then use Node.js 8.x or higher.
-
 ### Install the package
 
-Install the Azure Key Vault Admin client library using npm:
+Install the Azure Key Vault administration client library for JavaScript and TypeScript with [NPM][NPM]:
 
-`npm install @azure/keyvault-admin`
+```PowerShell
+npm install @azure/keyvault-admin
+```
 
-### Install the identity library
+### Prerequisites
 
-Key Vault clients authenticate using the Azure Identity Library. Install it as well using npm
+* An [Azure subscription][azure-sub].
+* An existing Azure Key Vault. If you need to create an Azure Key Vault, you can use the [Azure CLI][azure-cli].
+* If you are using this package in a Node.js application, then use Node.js 8.x or higher.
 
-`npm install @azure/identity`
+See the final two steps in the next section for details on creating the Key Vault with the Azure CLI.
```

Comment:
I would recommend linking to the right section from the second bullet point above which talks about creting key vault using azure cli instead of a separate note like this

Reply:
I believe I fixed this! Please review again.